Map/src/main/java/dk/gruppeseks/bodtrd/map/MapGenerator.java

```diff
@@ -77,12 +72,11 @@ public static void generateMap(List<Entity> walls, GameData gameData)
         {
             while (true)
             {
-                pos.setX((_rand.nextInt(mapGridX) + 1) * WALL_SIZE);
-                pos.setY((_rand.nextInt(mapGridY) + 1) * WALL_SIZE);
-
+                pos.setX((_rand.nextInt(mapGridX)) * wallSize);
```

Comment:
are you sure this is correct?
i think the +1 was actually supposed to be inside the ``_rand.nextInt()`` call as its range is [0;mapGridX[ ie X can never be == mapGridX